src/nfa/operations.cc

```diff
@@ -558,6 +581,7 @@ std::pair<Run, bool> Mata::Nfa::get_word_for_path(const Nfa& aut, const Run& run
 }
 
 //TODO: this is not efficient
+//method?
```

Comment:
If a method, it should be a method of `Run`. Otherwise, we could make it a method of `Nfa` by calling it something like `Nfa::Nfa::accepts(Run run)` or `Nfa::Nfa::has_in_lang(Run run)`. Calling the function otherwise reads as *NFA is in lang of run* which does not make any sense to me.

Reply:
Nice names could be accepts and accepts_prefix? It is however strange to give a run as a parameter, it should be a word. But it could return a run in fact, or have an optional parameter which where the run is put.

Reply:
I agree with both suggestions. `accepts[_...]` sounds good to me.

Reply:
It's probably a long abandoned relict of sad past that is already surpassed by the `Run` structure anyway. Is it even used?